### PR TITLE
NR S05a_01 Enhance dialogue between Tallin and Abhai

### LINKED ISSUE #6608: NR In Pursuit - "pales if possible"
### Language

English

### Source

NR In Pursuit

### Original Text

In the loyal Wraith's dialog upon being found, it is said he "(pales if possible)". That is commentary, not fact. You can't use figures-of-speech when describing facts in the third person. 

### Proposed Change

The way to get this dramatic effect is to have the person conversing with the Wraith to say that, either to the wraith, e.g. "The sound of Melifor's name seems to turn you even more pale, if that's at all possible" or to him/herself ("(Hmm, the sound of Melifor's name seems to have made this ghastly apparition even more pale, if that is at all possible)").

### PR DESCRIPTION
**Pull request by Descacharrado**
Squash before merging.

Closes #6608

### CODE REVIEW

**Review by nemaara — CHANGES_REQUESTED** [review]
Better to not make a concise line of dialogue into 3 that we have to click through. "Pales if possible" is definitely wrong but you can change that in a number of other ways as you see fit.

File: data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:2132]: Still a bit wordy, maybe something like this is better: 

```suggestion
            message= _ "<i>Is it just me or did this spirit get even paler?</i>  Can you tell us where he is? And, if I may ask, who are — or rather, were — you?"
```

Comment (Descacharrado) [reply data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:2132]: Ah, I understand, in that case I would reduce it even further, to something like:

`Did this spirit get even paler?`

**Review by nemaara — APPROVED** [review]

**Comment by stevecotton** [comment]
I see it's approved, is there anything left to do before merging?

String-freeze for 1.16.3 will be on Saturday, so if it's going to be backported then we should do it soon.

**Comment by nemaara** [comment]
Not from me, it can be merged as long as that po comment is sufficient for you.

File: data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg
Comment (eyalroz) [reply data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:2133]: I would separate the two questions, not putting them on consecutive lines of the same message. One doesn't ask two questions in immediate sequence without a pause to ponder the first one.

Comment (eyalroz) [reply data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:2133]: Also, "pale spirit" is a bit confusing, it's alluding to some sort of symbolic/moral aspect rather than the physicality, i.e. the wraith's spirit is paler as opposed to the wraith being paler. I realize that's not what you meant, but that's what it sounds like.

Comment (stevecotton) [reply data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:2133]: Two questions in one line is odd, but I think it fits the flow here - Tallin first asks the question related what he's currently trying to do, and then realises that he needs to build rapport with the new-found source of information first.

Comment (eyalroz) [reply data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:2133]: Yes, but I'd let him have a piece of dialog to himself, then for the user to press next/ok, then for him to talk to somebody else. So there's pause.

Comment (Descacharrado) [reply data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:2133]: > I would separate the two questions, not putting them on consecutive lines of the same message. One doesn't ask two questions in immediate sequence without a pause to ponder the first one.

One of them is a thought, to himself, and that is marked by the italic font. In game, you would not read them back to back with the same voice, you would unconsciously separate them, because one question is in italics and the next is not. 

> Also, "pale spirit" is a bit confusing, as if it was alluding to some sort of symbolic/moral aspect rather than the physicality.

I think the context clears that up, as you were talking to a ghost, and there would not be a subject for that moral reference. (e.g. Tallin's units, himself, Abhai... would not be referenced, not even by context for that moral aspect to make sense). To clear things up, for that to happen, in my opinion, the subject should be there in a sentence similar to:

`Did this wraiths' spirit get even paler?`
`Did his spirit get even paler?`
`Did my men's spirit get even paler?`

And that does not happen.

Comment (stevecotton) [reply data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:2133]: Including the unspoken thought, there's three questions in this line.

Comment (Descacharrado) [reply data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:2133]: > Including the unspoken thought, there's three questions in this line.

Uhhhh, true, but that was an issue before aswell then, as the two spoken questions have not been changed. I'd say that would be something to be considered in another issue.

Comment (eyalroz) [reply data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:2133]: >  as you were talking to a ghost,

The adjective "pale" does not apply to the noun "ghost", IMHO.

As for @stevecotton 's comment - before, it was 2 questions, now it's 3. 

As for whether to file another issue - if we agree here on some text, we can always file another issue, that's not the point...

**Pull request merged**
